### PR TITLE
Add USAGE.md

### LINKED ISSUE #32: Add a USAGE.md file
Usage files are really nice when they exist in a repo, because it makes it really obvious where to look for information about how to use this tool. It can be frustrating when you expect this file to be there and it's not. 
Please add a USAGE.md file, you can see an example of this 
[here](https://github.com/sendgrid/sendgrid-python/blob/master/USAGE.md)

Please make sure you modify the file for this repo (hint: Most of this information should be in the README file)

### PR DESCRIPTION
**Pull request by geomars**
## What does this PR do?
Close #32

### Checklist
- [x]  I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x]  I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x]  I have added in line documentation to the code I modified

### Short description of what this PR does:
* Add USAGE.md
* edit examples/example.py to be python 3 compatible

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-python?pullRequest=34) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/smtpapi-python?pullRequest=34) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/smtpapi-python?pullRequest=34) it.</sub>

**Comment by geomars** [comment]
Still confused whether the initialization part in USAGE.md overlaps with in Quick Start in README.md when the issue #32 is to add USAGE.md and not edit README.md

File: examples/example.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Python 2/3 compatible codebase
+from __future__ import absolute_import, division, print_function
```
Comment (mbernier) [inline examples/example.py]: Why did you add this?

**Comment by mbernier** [comment]
This is great work - I am just confused about that __future__ statement, as I am not sure it's necessary.

File: examples/example.py (hunk 1 above)
Comment (geomars) [reply examples/example.py]: The original last line on examples/example.py is print header.json_string() instead of print (header.json_string()) . I thought it might be intentional not an error or typho to keep the python2  version instead of moving to python3 by just change to print().  `from __future__ import absolute_import, print_function` enable the scripts backward compatibility with python2 while moving to python3 version.

Comment (mbernier) [reply examples/example.py]: NEAT!!

**Comment by thinkingserious** [comment]
Hello @geomars,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX
